imglib.py

```diff
@@ -49,13 +46,11 @@ def get_rotate(orientation):
         return 270
     elif orientation == 8:
         return 90
-    return 0
 
 
-def operations_run1(required_img, saving_path, img_size):
-    image_obj = Image.open(required_img)
-    orientation = get_orientation(required_img)
-    degrees = get_rotate(orientation)
+def saving_img(image_obj, path, img_size):
```

Comment:
Wrong function name. This function do:
- image rotation
- image resizing
- saving

Rename it pozhalusto.